governance/hardhat.config.js

```diff
@@ -64,3 +72,12 @@ module.exports = {
     enabled: true,
   },
 }
+
+if (process.env.ZK_SYNC) {
```

Comment:
Possible to do a single `if (process.env.ZK_SYNC)` with the includes above, for cleanliness?

Reply:
yes done